commands/curation/curationaudit.go

```diff
@@ -276,11 +282,17 @@ func (ca *CurationAuditCommand) getAuditParamsByTech(tech coreutils.Technology)
 			SetNpmOverwritePackageLock(true)
 	case coreutils.Maven:
 		ca.AuditParams.SetIsMavenDepTreeInstalled(true)
+	case coreutils.Pip:
+		if ca.PipRequirementsFile() == "" {
+			ca.SetPipRequirementsFile("requirements.txt")
+		}
 	}
+
 	return ca.AuditParams
 }
 
 func (ca *CurationAuditCommand) auditTree(tech coreutils.Technology, results map[string][]*PackageStatus) error {
+	start := time.Now()
 	flattenGraph, fullDependenciesTrees, err := audit.GetTechDependencyTree(ca.getAuditParamsByTech(tech), tech)
```

Comment:
should return the downlodUrls as return value

Reply:
commented in slack

Reply:
Fixed